includes/core-changes/msbuild/3.0/resource-manifest-name.md

```diff
@@ -1,78 +1,35 @@
-### Resource manifest file names
+### Resource manifest file name change
 
-Starting in .NET Core 3.0, the generated resource manifest file name has changed.
+Starting in .NET Core 3.0, in the default case, MSBuild generates a different manifest file name for resource files.
 
 #### Version introduced
 
 3.0
 
 #### Change description
 
-Prior to .NET Core 3.0, when [DependentUpon](/visualstudio/msbuild/common-msbuild-project-items#compile) metadata was set for a resource (*.resx*) file in the MSBuild project file, the generated manifest name was *Namespace.Classname.resources*. When [DependentUpon](/visualstudio/msbuild/common-msbuild-project-items#compile) was not set, the generated manifest name was *Namespace.Classname.FolderPathRelativeToRoot.Culture.resources*.
+Prior to .NET Core 3.0, if no `LogicalName`, `ManifestResourceName`, or `DependentUpon` metadata was specified for an `EmbeddedResource` item in the project file, MSBuild generated a manifest file name in the pattern `<RootNamespace>.<ResourceFilePathFromProjectRoot>.resources`. If `RootNamespace` is not defined in the project file, it defaults to the project name. For example, the generated manifest name for a resource file named *Form1.resx* in the root project directory was *MyProject.Form1.resources*.
 
-Starting in .NET Core 3.0, when a *.resx* file is colocated with a source file of the same name, for example, in Windows Forms apps, the resource manifest name is generated from the full name of the first type in the source file. For example, if *Type.cs* is colocated with *Type.resx*, the generated manifest name is *Namespace.Classname.resources*. However, if you modify any of the attributes on the `EmbeddedResource` property for the *.resx* file, the generated manifest file name may be different:
+Starting in .NET Core 3.0, if no `LogicalName`, `ManifestResourceName`, or `DependentUpon` metadata is specified for an `EmbeddedResource` item in the project file, and if the resource file is colocated with a source file of the same name (for example, *Form1.resx* and *Form1.cs*), MSBuild generates a manifest file name in the pattern `<Namespace>.<ClassName>.resources`, where the namespace and class name are extracted from the first type in the colocated source file. For example, the generated manifest name for a resource file named *Form1.resx* that's colocated with a source file named *Form1.cs* is *MyNamespace.Form1.resources*. The key thing to note is that the first part of the file name is different (*MyNamespace* instead of *MyProject*).
```

Comment:
The first sentence here is really long, could you split it up? Also, should `colocated` be hyphenated, `co-located`?

Reply:
The [style guide](https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=28765) says not to hyphenate words with prefixes unless omitting the hyphen could confuse the reader.